styleguide/source/_patterns/01-molecules/05-navigation/00-primary-nav/primary-nav.twig

```diff
@@ -1,134 +1,124 @@
-<div class="nav-primary-menu_button">Menu</div>
+<div class="nav-primary_button">Menu</div>
 
-<nav id="nav-primary" class="nav-primary-menu" role="navigation">
+<nav class="nav-primary" role="navigation" aria-label="Main Navigation">
 
-  <ul class="nav-primary_list nav-primary_list-closed" role="menubar">
-    <li class="nav-primary_list-item">
-      <div class="nav-primary_list-item_title">
-        {% include '04-link-primary-nav.twig' with { 'content': nav.first, 'url': '/patterns/04-pages-00-primary-landing-page-education/04-pages-00-primary-landing-page-education.html' } %}
-      </div>
+  <ul class="nav-primary_list nav-primary_list-mobile-closed" role="menubar">
+    <li class="nav-primary_section">
+        {% include '04-link-primary-nav.twig' with { 'content': nav.first } %}
```

Comment:
If you pass a url in to the `04-link-primary-nav.twig` instead of just the dummy `#` I'm seeing the anchor behavior take priority and navigate the user away instead of expanding the menu. We may need some `e.preventDefault()` methods in the js.